src/app/users/views/selectManageConsoleRoles.ejs

```diff
@@ -0,0 +1,54 @@
+<div class="govuk-width-container">
+  <div class="col-8">
+    <h1 class="govuk-heading-xl">
+        <%= addOrChangeService ? 'Change Roles' : 'Add roles' %>
+    </h1>
+    <table class="govuk-table">
+        <tbody class="govuk-table__body">
+          <tr class="govuk-table__row">
+            <th scope="row" class="govuk-table__header"><b>User</b></th>
+            <td class="govuk-table__cell"><%= locals.user.name %></td>
+          </tr>
+          <tr class="govuk-table__row">
+            <th scope="row" class="govuk-table__header"><b>Service</b></th>
+            <td class="govuk-table__cell"><%= serviceSelectedByUser.name %></td>
+        </tr>
+        </tbody>
+      </table>
+      <form method="post">
+        <input type="hidden" name="_csrf" value="<%=csrfToken%>" />
+          <div class="govuk-form-group">
+            <fieldset class="govuk-fieldset" aria-describedby="waste-hint" id="roleSelection">
```

Comment:
Could we rename `waste-hint` to `role-hint` or similar? On this line and the one below it?